G1ANT.Addon.Net/G1ANT.Addon.Net/Variables/TimeoutMailSmtpVariable.cs

```diff
@@ -9,7 +9,7 @@ namespace G1ANT.Addon.Net.Commands
 {
     [Variable(
         Name = "timeoutmailsmtp",
-        Tooltip = "Determines the timeout value (in ms) for the [mail.smtp](G1ANT.Addon/G1ANT.Addon.Net/G1ANT.Addon.Net/Commands/MailSmtpCommand.md) command; the default value is 10000 (10 seconds).")]
+        Tooltip = "Determines the timeout value (in ms) for the mail.smtp command; the default value is 10000 (10 seconds)")]
```

Comment:
Determines the timeout value (in ms) for the `mail.smtp` command; the default value is 10000 (10 seconds)